helm/endpoint-copier-operator/templates/rbac/role.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
```

Comment:
Should we also add the `{{- include "endpoint-copier-operator.labels" . | nindent 4 }}` label here? It is added for the `Role`, `RoleBinding` and `ClusterRoleBinding`?